ui/components/src/DateTimeRangePicker/TimeRangeSelector.tsx

```diff
@@ -42,15 +42,21 @@ export function TimeRangeSelector(props: TimeRangeSelectorProps) {
     <Select
       value={formattedValue}
       onChange={onSelectChange}
-      IconComponent={Calendar}
+      IconComponent={(props) => {
+        // "transform: none" prevents calendar icon from flipping over when menu is open
+        return <Calendar {...props} sx={{ transform: 'none !important' }} />;
+      }}
       inputProps={{
         'aria-label': `Select time range. Currently set to ${formattedValue}`,
       }}
       sx={{
         '.MuiSelect-icon': {
           marginTop: '1px',
         },
-        '.MuiSelect-select': height ? { lineHeight: height, paddingY: 0 } : {},
+        // "paddingRight" creates more space for the calendar icon (it's a bigger icon)
+        '.MuiSelect-select': height
+          ? { lineHeight: height, paddingY: 0, paddingRight: '36px !important' }
```

Comment:
Any way to get more specific with the `.MuiSelect-select` class so !important isn't needed or I'm assuming MUI just doesn't give a helpful class here?

Reply:
I'll give it a shot, let's see!

Reply:
`!important` is not the best things to use if it can be avoided :)

Reply:
@sjcobb That works much better, thank you